ironic/drivers/modules/drac/inspect.py

```diff
@@ -44,7 +93,261 @@ class DracRedfishInspect(redfish_inspect.RedfishInspect):
     specific incompatibilities and introduction of vendor value added
```

Comment:
Can you remove lines 91-94 since we are now providing custom code here?

Reply:
Done